ovs/extensions/log.py

```diff
@@ -38,66 +39,277 @@
 from requests.packages.urllib3.exceptions import SNIMissingWarning
 
 
+# Handlers
+OVS_REDIS_HANDLER = 'ovs_redis'
+OVS_STREAM_HANDLER = 'ovs_console'
+OVS_FILE_HANDLER = 'ovs_file_{0}'  # Format is the name of the associated logger
+OVS_FORMATTER_NAME = 'ovs'
+
 # Configures the logger
-DEFAULT_LOGGER_CONFIG = {'handlers': ['default'],
-                         'level': 'INFO',
-                         'propagate': True}
-
-DEFAULT_LOG_HANDLER_CONFIG = {'default': {'level': 'INFO',
-                                          'class': 'logging.StreamHandler',
-                                          'formatter': 'ovs'}}
-
-DEFAULT_LOG_CONFIG = {'version': 1,
-                      'disable_existing_loggers': False,
-                      'formatters': {
-                          'standard': {'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'},
-                          'ovs': OVS_FORMATTER_CONFIG
-                      },
-                      'handlers': DEFAULT_LOG_HANDLER_CONFIG,
-                      'loggers': {OVS_LOGGER: DEFAULT_LOGGER_CONFIG,
-                                  EXTENSIONS_LOGGER_NAME: DEFAULT_LOGGER_CONFIG,
-                                  API_LOGGER: DEFAULT_LOGGER_CONFIG,
-                                  'urllib3': {'level': 'WARNING'},
-                                  'paramiko': {'level': 'WARNING'},
-                                  'requests': {'level': 'WARNING'}}}
+LIBRARY_LOG_CONFIGS = {'urllib3': {'level': 'WARNING'},
```

Comment:
Is it necessary to put this into a variable?

Reply:
Just to separate the different loggers